web/js/modules/settings/actions.js

```diff
@@ -29,3 +30,15 @@ export function changeDatelineVisibility(value) {
     value,
   };
 }
+
+export function changeCoordinateFormat(value) {
+  if (!value) {
```

Comment:
Is there a case where we don't pass a `value` to this action creator?  I think there always has to be _some_ coordinate format to reference so, maybe we don't need the conditional and only need the `setItem()` call. 🤔

Reply:
You're right, I was able to remove the conditional statement as there are no situations where we are not passing a value to the action creator.